openedx/features/caliper_tracking/tests/current/edx.forum.thread.created.json

```diff
@@ -0,0 +1,41 @@
+{
+  "accept_language": "en-US,en;q=0.5",
+  "agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:62.0) Gecko/20100101 Firefox/62.0",
+  "context": {
+    "course_user_tags": {},
+    "course_id": "course-v1:edX+DemoX+Demo_Course",
```

Comment:
Kindly sort that portion.
course_id should come first.

Reply:
done